_includes/head.html

```diff
@@ -13,4 +13,8 @@
   {% if jekyll.environment == 'production' and site.google_analytics %}
   {% include google-analytics.html %}
   {% endif %}
+
+  {% if site.gems contains 'jekyll-seo-tag' %}
+  {% include seo.html %}
```

Comment:
Documentation of *this particular* plugin says to install via `site.gems`, so I'd say the how to install a plugin documentation is irrelevant as here, yeah you can have plugins in _plugins but that doesn't apply to the gem `jekyll-seo-tag`, it's own installation instructions should take precedence here.